cluster-agent/README.md

```diff
@@ -57,20 +94,21 @@ helm install appdynamics-charts/cluster-agent --name=stable \
 | `deployment.image.tag` | Tag of the machine agent image | `latest`
 | `deployment.image.pullPolicy` | The machine agent image pull policy| `IfNotPresent`
 | `deployment.resources ` | Resources directive  | See below
-
+| `existingSecret` | Name of existing secret to use for ClusterAgent application in AppDyanamics   |
 
 Example resource limits:
 
 ```
   resources:
     limits:
       cpu: 200m
-      memory: "300M"
-    requests: 
+      memory: "300Mi"
+    requests:
       cpu: 100m
-      memory: "100M"
+      memory: "100Mi"
  ```
+## Limitation
+A fully functional AppDyanmics account is needed to install this chart. Lite Appdyanamics account is not currently supported.
 
-
-
- 
+## Support
```

Comment:
This section is a mandatory section to be included in the ReadMe for charts in the catalog.
https://github.com/IBM/charts/blob/master/GUIDELINES.md#support-statement